app/components/location-map.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  maps: Ember.inject.service(),
+
+  didInsertElement() {
+    this._super(...arguments);
+    Ember.Logger.debug('didInsertElement');
+    this.$('.map-container').append(this.get('maps').getMapElement(this.get('location')));
```

Comment:
Given that idiomatic Ember tends to use the service name like we've got here, I would be inclined to leave as is (but maybe add a code comment explaining what is going on above this line?)